### DISCUSSION TITLE
Ollama

### DISCUSSION THREAD

**Discussion by inluxc** [coollabsio/coolify#1373] [Service Template Requests]
It would be great do add an AI open source service like Ollama.

https://github.com/jmorganca/ollama

**Comment by stefan520** [comment]
Oh yes. Thanks

**Comment by its-aye-lisa** [comment]
This exists now

**Reply by LunaVoid** [reply]
Not fully, what's currently included is just Ollama with the Open Web UI. If you want to run Ollama by itself, you can remove the Web UI from the Docker Compose setup. That said, there's rarely a reason to run the Ollama server standalone, since it shouldn't be directly exposed to the internet.

**Reply by its-aye-lisa** [reply]
Oh, goooood to know!

And true ... @inluxc do you feel satisfied now?
